csrc/serde/fusion_record_serde.cpp

```diff
@@ -395,32 +395,15 @@ void RecordFunctorFactory::registerAllParsers() {
   auto deserializeBroadcastInDimRecord =
```

Comment:
Consolidating serde deserialization of `broadcast_in_dim` into just one Record Type given that the template argument for `shape` was removed.